src/pages/collab/index.tsx

```diff
@@ -45,13 +47,15 @@ const MatchRequestPage = () => {
   const { data: numOfMatchRequests } =
     api.matchRequest.getNumOfMatchRequests.useQuery();
 
-  const { mutate: notifyOnAutomaticMatchedRequests } =
-    api.matchRequest.notifyOnAutomaticMatchedRequests.useMutation();
+  const { mutate: automaticallyMatchCurrentUserRequest } =
```

Comment:
Whats the point in renaming this

Reply:
heyo, i tried to describe what i understood of the automatic matching process in the pr description, if correct then this name more accurately reflects what it is doing. 

the role isnt to notify but rather to try to automatically match it. similar to the other mutations have event emitters the notification is a byproduct.

Reply:
it's role is to notify tho